go.mod

```diff
@@ -4,32 +4,23 @@ go 1.13
 
 require (
 	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
-	github.com/gogo/protobuf v1.3.1 // indirect
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/golang/groupcache v0.0.0-20191027212112-611e8accdfc9 // indirect
 	github.com/googleapis/gnostic v0.3.1 // indirect
 	github.com/hashicorp/golang-lru v0.5.3 // indirect
 	github.com/imdario/mergo v0.3.8 // indirect
+	github.com/json-iterator/go v1.1.9 // indirect
 	github.com/nginxinc/nginx-plus-go-client v0.6.0
 	github.com/nginxinc/nginx-prometheus-exporter v0.5.1-0.20200122150314-eb956e2ca62e
-	github.com/onsi/ginkgo v1.10.1 // indirect
-	github.com/onsi/gomega v1.7.0 // indirect
 	github.com/prometheus/client_golang v1.3.0
-	github.com/spf13/pflag v1.0.5 // indirect
-	golang.org/x/crypto v0.0.0-20191108234033-bd318be0434a // indirect
-	golang.org/x/net v0.0.0-20191112182307-2180aed22343 // indirect
-	golang.org/x/sys v0.0.0-20200122134326-e047566fdf82 // indirect
+	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e // indirect
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
-	golang.org/x/tools v0.0.0-20191113055240-e33b02e76616 // indirect
+	golang.org/x/tools v0.0.0-20200325203130-f53864d0dba1 // indirect
 	google.golang.org/appengine v1.6.5 // indirect
-	gopkg.in/inf.v0 v0.9.1 // indirect
-	gopkg.in/yaml.v2 v2.2.5 // indirect
-	k8s.io/api v0.0.0-20191016110408-35e52d86657a
-	k8s.io/apimachinery v0.0.0-20191004115801-a2eda9f80ab8
-	k8s.io/client-go v0.0.0-20191016111102-bec269661e48
-	k8s.io/code-generator v0.0.0-20191004115455-8e001e5d1894
-	k8s.io/gengo v0.0.0-20191108084044-e500ee069b5c // indirect
-	k8s.io/klog v1.0.0 // indirect
-	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a // indirect
-	k8s.io/utils v0.0.0-20191030222137-2b95a09bc58d // indirect
+	k8s.io/api v0.18.0
```

Comment:
some tags are `v0.18.0`, some are `v0.18.1-beta.0`. Was it intentional? 🤔

Reply:
@pleshakov this was the result of:
```sh
go get -u "k8s.io/client-go@kubernetes-1.18.0"
go get -u "k8s.io/code-generator@kubernetes-1.18.0"
go get -u "k8s.io/apimachinery@kubernetes-1.18.0"
go get -u "k8s.io/api@kubernetes-1.18.0"
```

🤔 


Seems to just be a symbolic link/alias for the same release?:
E.g: https://github.com/kubernetes/code-generator/tags